pkg/workflow/status.go

```diff
@@ -123,8 +124,14 @@ func (m *StatusSyncer) SyncWorkflowExecutionStatus(ctx context.Context, wf *wfv1
 		m.Logger.Debugf("workflow %s is succeeded", we.ID.String())
 
 	case wfv1.WorkflowFailed, wfv1.WorkflowError:
-		status.WorkflowExecutionStatusPending.True(we, "")
-		status.WorkflowExecutionStatusRunning.False(we, wf.Status.Message)
+		switch {
+		case status.WorkflowExecutionStatusStopped.IsUnknown(we):
+			status.WorkflowExecutionStatusStopped.True(we, wf.Status.Message)
```

Comment:
What does the message look like in this case

Reply:
The message here will be `Stopped with strategy 'Terminate'`

Reply:
Let's using gray color to represent `stopped` status.